public/app/plugins/panel/timeseries/measureDataRenderDelay.ts

```diff
@@ -0,0 +1,42 @@
+import { closestIdx, DataFrame, FieldType } from '@grafana/data';
+import { reportPerformance } from 'app/core/services/echo/EchoSrv';
+import { PerformanceEventName } from 'app/core/services/echo/backends/PerformanceBackend';
+
+const measureDataRenderDelayForFrame = (prevFrame: DataFrame, newFrame: DataFrame, now: number) => {
```

Comment:
can you add a comment and simple unit test that illustrates what this is actually capturing.  I *think* it is capturing the time between rendered frames

Reply:
added unit tests https://github.com/grafana/grafana/blob/35b1acedca129b67883a5c7811935af454e9b9cf/public/app/plugins/panel/timeseries/measureDataRenderDelay.test.ts#L43-L48